adapter/internal/discovery/xds/server.go

```diff
@@ -846,6 +846,21 @@ func RemoveAPICacheForEnv(adapterInternalAPI model.AdapterInternalAPI, envType s
 	}	
 }
 
+// RemoveAPIFromOrgAPIMap removes api from orgAPI map
+func RemoveAPIFromOrgAPIMap(adapterInternalAPI model.AdapterInternalAPI, orgId string) {
```

Comment:
you just use the uuid from the adapterInternalAPI right? why not directly send it here?

Reply:
Modified the code